### PR TITLE
docs: fix Database Utility Class `getXMLFromResult()`

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- fix `getXMLFromResult()` description
- add "Initializing the Utility Class"

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Comment by kenjis** [comment]
See #8277

File: user_guide_src/source/database/utilities/001.php [hunk 1]
```diff
@@ -1,13 +1,15 @@
 <?php
 
-class MyModel extends \CodeIgniter\Model
-{
-    protected $table      = 'foo';
-    protected $primaryKey = 'id';
-}
+$db     = db_connect();
+$dbutil = \CodeIgniter\Database\Config::utils();
```
Comment (paulbalandan) [inline user_guide_src/source/database/utilities/001.php:4]: Can we use `Config\Database::utils()` instead?

File: user_guide_src/source/database/utilities/002.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+$dbutil = \CodeIgniter\Database\Config::utils();
```
Comment (paulbalandan) [inline user_guide_src/source/database/utilities/002.php:3]: `Config\Database`?

File: user_guide_src/source/database/utilities/001.php (hunk 1 above)
Comment (kenjis) [reply user_guide_src/source/database/utilities/001.php:4]: Oh, thanks! Fixed

File: user_guide_src/source/database/utilities/002.php (hunk 2 above)
Comment (kenjis) [reply user_guide_src/source/database/utilities/002.php]: Fixed.

**Pull request merged**